src/libmongoc/tests/json/crud/unified/find-allowdiskuse.json

```diff
@@ -32,7 +32,7 @@
   ],
   "tests": [
     {
-      "description": "Find does not send allowDiskuse when value is not specified",
+      "description": "Find does not send allowDiskUse when value is not specified",
```

Comment:
Also pulled in while syncing the entire `crud/unified` directory. I'm not sure why this wasn't included in [CDRIVER-4358](https://jira.mongodb.org/browse/CDRIVER-4358) (since resolved), but that would have been the relevant issue. Likely just an oversight, as it was only a description change (`aggregate-allowdiskuse.json` was the actual new test file there).